release/ray_release/byod/requirements_ml_byod_3.8.txt

```diff
@@ -679,16 +684,26 @@ cython==0.29.35 \
     --hash=sha256:ef2fc6f81aa8fb512535b01199fbe0d0ecafb8a29f261055e4b3f103c7bd6c75 \
     --hash=sha256:fb8c11cd3e2d5ab7c2da78c5698e527ecbe469437326811562a3fbf4c5780ae4
     # via mujoco-py
+datasets==2.14.1 \
```

Comment:
Is this the new way to write a release test? We need to keep updating this `requirements_ml_byod*.txt` file? 
How can we get these hash and "via" info?

Reply:
Hi yes this is the new way to specify dependencies for release tests. I'll send an email to engineers about this soon.

You don't actually change this file. You add the package name to release/ray_release/byod/requirements_ml_byod_3.8.in

and compile this file with bazel run //release:requirements_ml_byod_3.8.update